homeassistant/components/hassio/ingress.py

```diff
@@ -294,7 +297,7 @@ async def _websocket_forward(ws_from, ws_to):
             elif msg.type == aiohttp.WSMsgType.PONG:
                 await ws_to.pong()
             elif ws_to.closed:
-                await ws_to.close(code=ws_to.close_code, message=msg.extra)
+                await ws_to.close(code=ws_to.close_code, message=msg.extra)  # type: ignore[arg-type]
```

Comment:
This looks like it might be a bug as `msg.extra` is `str` and `message` is supposed to be `bytes`

but it works because `WebSocketReader.close` converts it
```
    async def close(self, code: int = 1000, message: Union[bytes, str] = b"") -> None:
        """Close the websocket, sending the specified code and message."""
        if isinstance(message, str):
            message = message.encode("utf-8")
```

Maybe we need to fix the typing on `close` in `aiohttp`